code/controllers/master.dm

```diff
@@ -264,8 +264,8 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 
 		var/ss_runlevels = SS.runlevels
 		var/added_to_any = FALSE
-		for(var/I in 1 to GLOB.bitflags.len)
-			if(ss_runlevels & GLOB.bitflags[I])
+		for(var/I in 1 to GLOB.runlevel_flags.len)
+			if(ss_runlevels & GLOB.runlevel_flags[I])
```

Comment:
You're optimizing this loop at the expense of requiring the runlevels list to be explicitly defined somewhere outside of the MC. So, now in order to add a new runlevel, you need to remember to add it to said list. Given that this is generally only run once at init and it's only a maximum of 16(?) iterations, it creates more maintainability issues than it improves performance